### ISSUE TITLE
Support Ukrainian language recognition

### ISSUE THREAD

**Issue by Vladykus** [C-Nedelcu/talk-to-chatgpt#103]
Could you please add Ukrainian language text to speech. In one chrome extension they may translate it to after checking captcha.
https://cloud.google.com/text-to-speech
If there is use Ukrainian language, could you please add it to the ChatGPT text to speech.

https://texttospeech.googleapis.com/v1beta1/text:synthesize
"languageCode": "uk-UA",
    "name": "uk-UA-Wavenet-A"
https://cloud.google.com/text-to-speech/docs/apis

**Comment by C-Nedelcu** [comment]
Hello Vladykus,

Thank you for your message. Currently, we don't use the "cloud" Google thing, but we use the web browser capabilities. 
Have you tried Microsoft Edge? maybe they have support for Ukrainian?

We implemented a third party TTS solution called ElevenLabs. Sadly they do not have Ukranian, I believe. But if there exists another good TTS API , with javascript support, and with good quality & lots of languages, we could consider another integration?

**Comment by Vladykus** [comment]
> 

Yeah, I know ElevenLabs and almost all of other translators doesn't have Ukrainian language, except a few of corporate programs like Microsoft Azura. Unfortunately I didn't found any good Ukrainian TTS except one is included in Google Chrome App called Read Aloud, that is why I thought that if there is Ukrainian language supported, maybe you could add it to your app as well.
